pkg/sypgp/sypgp.go

```diff
@@ -30,6 +31,12 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// PublicKeyType is the armor type for a PGP public key.
+var PublicKeyType = "PGP PUBLIC KEY BLOCK"
+
+// PrivateKeyType is the armor type for a PGP private key.
+var PrivateKeyType = "PGP PRIVATE KEY BLOCK"
```

Comment:
Why is this a `var`? Can it be a `const`?

```suggestion
const PrivateKeyType = "PGP PRIVATE KEY BLOCK"
```